code/modules/mob/living/simple_animal/hostile/gremlin.dm

```diff
@@ -1,5 +1,5 @@
 //Gremlins
-//Small monsters that don't usually attack humans or other animals. Instead they mess with electronics, computers and machinery
+//Small monsters that don't attack humans or other animals. Instead they mess with electronics, computers and machinery
```

Comment:
Which is why I said "get into the habit", and not "take this change out of this PR".